microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -77,6 +77,26 @@ When you visit a webpage and submit a form, Microsoft Edge sends information abo
 
 In **Guest** mode, autofill is not available and new autofill entries aren't added.  For **InPrivate** mode, Microsoft Edge offers autofill entries, but new autofill entries aren't added.
 
+<!-- ====================================================================== -->
+## Bing Chat
```

Comment:
```suggestion
## Bing chat
```
style, look-and-feel decision: considerations:
* corp style guide: for headings, use sentence case (it would be 'chat').
* In UI capture in [the section](https://github.com/MicrosoftDocs/edge-developer/blob/user/majenny/ComposePrivacy/microsoft-edge/privacy-whitepaper/index.md#bing-chat), it's **Chat**. 
* In body of section, it's "Bing chat".